lib/modules.nix

```diff
@@ -338,7 +338,7 @@ rec {
     # Type-check the remaining definitions, and merge them.
     mergedValue = foldl' (res: def:
       if type.check def.value then res
-      else throw "The option value `${showOption loc}' in `${def.file}' is not a ${type.description}.")
+      else throw "The option value `${showOption loc}' in `${def.file}' is not of type `${type.description}'.")
```

Comment:
From what I understand we are moving away from asymmetrical quoting?
This set of commit changes the quoting within comments as well, please revert to do only one type of quotation mark.

Reply:
We are? My impression was Eelco would like to keep the “special” quoting?

Reply:
```
12:03:46        niksnut | using `...' is always wrong
```

heh. :)

Reply:
Unicode quotes have practical implications that make them undesirable.

See https://github.com/NixOS/rfcs/pull/4

Reply:
` is not unicode, though.